tests/SprykerEcoTest/Zed/UnzerRestApi/Communication/Plugin/CheckoutRestApi/UnzerCheckoutDataExpanderPluginTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEcoTest\Zed\UnzerRestApi\Business\UnzerRestApiFacade;
+
+use Codeception\TestCase\Test;
+use Generated\Shared\DataBuilder\RestCheckoutDataBuilder;
+use Generated\Shared\DataBuilder\UnzerCredentialsBuilder;
+use Generated\Shared\Transfer\QuoteTransfer;
+use Generated\Shared\Transfer\RestCheckoutRequestAttributesTransfer;
+use SprykerEco\Zed\UnzerRestApi\Communication\Plugin\CheckoutRestApi\UnzerCheckoutDataExpanderPlugin;
+
+/**
+ * Auto-generated group annotations
+ *
+ * @group SprykerEcoTest
+ * @group Zed
+ * @group UnzerRestApi
+ * @group Business
+ * @group UnzerRestApiFacade
+ * Add your own group annotations below this line
+ */
+class UnzerCheckoutDataExpanderPluginTest extends Test
+{
+    /**
+     * @return void
+     */
+    public function testExpandCheckoutDataShouldExpandWhileUnzerCredentialsAreSpecifiedWithinQuote(): void
+    {
+        // Arrange
+        $unzerCredentailsTransfer = (new UnzerCredentialsBuilder())->withUnzerKeypair()->build();
+        $restCheckoutDataTransfer = (new RestCheckoutDataBuilder())->withQuote([
+            QuoteTransfer::UNZER_CREDENTIALS => $unzerCredentailsTransfer->toArray(),
+        ])->build();
+
+        // Act
+        $restCheckoutDataTransfer = $this->tester
+            ->getFacade()
+            ->expandCheckoutData($restCheckoutDataTransfer);
+
+        // Assert
+        $this->assertSame(
+            $unzerCredentailsTransfer->getUnzerKeypair()->getPublicKey(),
+            $restCheckoutDataTransfer->getQuote()->getUnzerCredentials()->getUnzerKeypair()->getPublicKey(),
+        );
+    }
+
+    /**
+     * @return void
+     */
+    public function testExpandCheckoutDataShouldNotExpandWhileUnzerCredentialsAreNotSpecifiedWithinQuote(): void
```

Comment:
* We don't need to test the plugin if it delegates the call to the module's facade, client, or service.

Reply:
Can stay as discussed